src/projectConfig.js

```diff
@@ -24,6 +24,11 @@ export default {
         extras: { $ref: '#/definitions/extras' },
         metadata: { $ref: '#/definitions/metadata' },
         sync: { $ref: '#/definitions/sync' },
+        // Project Name
```

Comment:
the project name is already encoded in the key that holds this object - why would we want to duplicate it here?